packages/react-native-bpk-component-chip/src/BpkChipWrapper.js

```diff
@@ -120,39 +148,52 @@ const BpkChipWrapper = (props: Props) => {
     iconStyle.push(styles.iconDisabled);
   }
 
+  if (type === CHIP_TYPES.outline) {
+    userStyle.push({ elevation: 0 });
+    innerStyle.push(styles.innerOutline);
+    textStyle.push(styles.textOutline);
+    iconStyle.push(styles.iconOutline);
+
+    if (selected) {
+      innerStyle.push(styles.innerOutlineSelected);
+      textStyle.push(styles.textSelected);
```

Comment:
Shouldn't this be done regardless of `CHIP_TYPE`?

Reply:
Refactored!